src/include/misc_inline.h

```diff
@@ -221,6 +221,34 @@ __wt_timing_stress(WT_SESSION_IMPL *session, u_int flag)
         __wt_sleep(0, i * (WT_TIMING_STRESS_MAX_DELAY / 10));
 }
 
+/*
+ * __wt_failpoint --
+ *     A generic failpoint function, it will return true if the failpoint triggers. Takes a double
+ *     representing the probability of the failpoint occurring. Supports percentages with two
+ *     decimal places.
+ */
+static inline bool
+__wt_failpoint(WT_SESSION_IMPL *session, uint64_t conn_flag, double probability)
+{
```

Comment:
This doesn't need a double, does it?

Reply:
Nah I just feel like double is more human readable but I will update it to not be an improve the comment so its clear what the variable is.

Reply:
Fixed in 8801a29